src/TypeSpec.Extension/Emitter.Csharp/src/lib/json-serialize-refs.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@ArcturusZhang :  I'm not 100% sure this is allowable.   Please reach out to our PM team and collaborate with them to engage the MS Open Source office for guidance.